src/test/scala/nl/knaw/dans/easy/stage/fileitem/EasyStageFileItemSpec.scala

```diff
@@ -48,4 +50,24 @@ class EasyStageFileItemSpec extends FlatSpec with Matchers {
     val args = "src/test/resources/example.csv outdir".split(" ")
     getSettingsRows(new FileItemConf(args)).get.size shouldBe 5
   }
+
```

Comment:
The failing travis build while the local build succeeds proofs your point. I'm currently considering extracting leaf elements of the expected output and checking there presence in the actual output.